x-pack/plugins/observability_solution/apm/public/components/app/service_inventory/multi_signal_inventory/index.tsx

```diff
@@ -172,45 +175,53 @@ export function MultiSignalInventory() {
-              defaultMessage: 'Search services by name',
-            })}
-            searchQuery={searchQuery}
-            onChangeSearchQuery={setSearchQuery}
-          />
-        </EuiFlexItem>
-        <EuiFlexItem grow={false}>
-          <SearchBar showQueryInput={false} />
-        </EuiFlexItem>
-      </EuiFlexGroup>
-      <EuiFlexGroup direction="column" gutterSize="m">
-        <EuiFlexItem>
-          <MultiSignalServicesTable
-            status={mainStatisticsStatus}
-            data={filteredData}
-            initialSortField={initialSortField}
-            initialPageSize={INITIAL_PAGE_SIZE}
-            initialSortDirection={INITIAL_SORT_DIRECTION}
-            timeseriesData={timeseriesDataFetch?.data}
-            timeseriesDataLoading={timeseriesDataFetch.status === FETCH_STATUS.LOADING}
-            noItemsMessage={
-              <EmptyMessage
-                heading={i18n.translate('xpack.apm.servicesTable.notFoundLabel', {
-                  defaultMessage: 'No services found',
+      {!data?.hasData && status === FETCH_STATUS.SUCCESS ? (
+        <NoEntitiesEmptyState />
+      ) : (
+        <>
+          <EuiFlexGroup gutterSize="m">
+            <EuiFlexItem grow>
+              <TableSearchBar
+                placeholder={i18n.translate('xpack.apm.servicesTable.filterServicesPlaceholder', {
+                  defaultMessage: 'Search services by name',
                 })}
+                searchQuery={searchQuery}
+                onChangeSearchQuery={setSearchQuery}
               />
-            }
-          />
-        </EuiFlexItem>
-      </EuiFlexGroup>
+            </EuiFlexItem>
+            <EuiFlexItem grow={false}>
+              <SearchBar showQueryInput={false} />
+            </EuiFlexItem>
+          </EuiFlexGroup>
+          <EuiFlexGroup direction="column" gutterSize="m">
+            <EuiFlexItem>
+              <MultiSignalServicesTable
+                status={mainStatisticsStatus}
+                data={filteredData}
+                initialSortField={initialSortField}
+                initialPageSize={INITIAL_PAGE_SIZE}
+                initialSortDirection={INITIAL_SORT_DIRECTION}
+                timeseriesData={timeseriesDataFetch?.data}
+                timeseriesDataLoading={timeseriesDataFetch.status === FETCH_STATUS.LOADING}
+                noItemsMessage={
+                  <EmptyMessage
+                    heading={i18n.translate('xpack.apm.servicesTable.notFoundLabel', {
+                      defaultMessage: 'No services found',
+                    })}
+                  />
+                }
+              />
+            </EuiFlexItem>
+          </EuiFlexGroup>
+        </>
+      )}
+      <Welcome
+        isModalVisible={tourState.isModalVisible ?? false}
+        onClose={() => setTourState({ ...tourState, isModalVisible: false })}
+        onConfirm={() => setTourState({ ...tourState, isModalVisible: false })}
```

Comment:
You could export a function to hide the modal and use it here. It's totally up to you though.
```suggestion
        onClose={hideModal}
        onConfirm={hideModal}
```